tcf_website/views/review.py

```diff
@@ -123,3 +123,19 @@ def get_object(self):  # pylint: disable=arguments-differ
             raise PermissionDenied(
                 "You are not allowed to delete this review!")
         return obj
+
+    def delete(self, request, *args, **kwargs):
+        """Overide DeleteView's delete function to add a message confirming deletion."""
+        # Note: we don't use SuccessMessageMixin because it currently has issues
+        # with DeleteViews.
+        # See:
+        # https://stackoverflow.com/questions/24822509/success-message-in-deleteview-not-shown
+        course = super().get_object().course
+
+        messages.add_message(
+            request,
+            messages.SUCCESS,
+            'Successfully deleted the review for ' +
```

Comment:
I think this should be `deleted your review` - what do you think?

Reply:
That's more concise and gets the message across better. I originally had something like this, but I used the "Successfully deleted the review for ..." phrasing to be consistent with your screenshots in #357, which both follow the format "Successfully created/updated/deleted the review for <class>."

That said, I prefer the more concise version and will implement that.

Reply:
Whoops, I meant `deleted your review for (course)`, as I think stating the course information to confirm is important to avoid having users worry about potentially deleting the wrong review. Regarding the screenshots in the issue I made, those were taken from a PR (#355) where I left the same feedback...my bad, should've made that clear. Sorry!